infrastructure/cluster/flux-v2/jhub/jhub-release.yaml

```diff
@@ -164,18 +165,22 @@ spec:
         RUCIO_WILDCARD_ENABLED: "1"
         RUCIO_BASE_URL: "https://vre-rucio.cern.ch"
         RUCIO_AUTH_URL: "https://vre-rucio-auth.cern.ch"
-        RUCIO_DISPLAY_NAME: "VRE-RUCIO"
+        RUCIO_WEBUI_URL: "https://vre-rucio-ui.cern.ch"
+        RUCIO_DISPLAY_NAME: "RUCIO - CERN VRE"
         RUCIO_NAME: "vre-rucio.cern.ch"
-        RUCIO_SITE_NAME: "ROAMING"
-        RUCIO_OIDC_AUTH: "env"
-        RUCIO_OIDC_ENV_NAME: "REFRESH_TOKEN"
+        RUCIO_SITE_NAME: "CERN"
+        RUCIO_OIDC_AUTH: "file"
+        RUCIO_OIDC_FILE_NAME: "/tmp/rucio_oauth.token"
         RUCIO_DEFAULT_AUTH_TYPE: "oidc"
         RUCIO_OAUTH_ID: "rucio"
         RUCIO_DEFAULT_INSTANCE: "vre-rucio.cern.ch"
         RUCIO_DESTINATION_RSE: "CERN-EOS"
         RUCIO_RSE_MOUNT_PATH: "/eos/escape"
         RUCIO_PATH_BEGINS_AT: "5"
         RUCIO_CA_CERT: "/certs/rucio_ca.pem"
+        RUCIO_VOMS_VOMSES_PATH: "/etc/vomses"
```

Comment:
Note: We could try to remove these voms env variables for troubleshooting if it doesn't work after all.

Reply:
As you prefer, I don't have strong opinions. I were adding them just for completeness

Reply:
Commenting them for not "forgetting" that we could add them